go.mod

```diff
@@ -57,13 +57,18 @@ require (
 	golang.org/x/time v0.0.0-20220609170525-579cf78fd858
 	google.golang.org/api v0.85.0
 	google.golang.org/grpc v1.47.0
-	google.golang.org/grpc/examples v0.0.0-20220622233350-5cdb09fa29c1
+	google.golang.org/grpc/examples v0.0.0-20220623204041-06ad0b82211b
 	gopkg.in/square/go-jose.v2 v2.6.0
 	gopkg.in/validator.v2 v2.0.1
 	gopkg.in/yaml.v3 v3.0.1
 	modernc.org/sqlite v1.17.3
 )
 
+require (
```

Comment:
I thought we were trying to keep to two sections?

Reply:
![image](https://user-images.githubusercontent.com/2232524/175422869-88bc1373-3111-45cc-8b27-ab5794adb2c5.png)